### DISCUSSION TITLE
How to generate CustomResourceDefinition from struct definition?

### DISCUSSION THREAD

**Discussion by ivaltryek** [kube-rs/kube#1413] [General]
Is there a way to generate CRD from struct rather writing it manually? I've seen that Kubebuilder allows it in go. 

Ref: https://book.kubebuilder.io/reference/generating-crd

**Comment by clux** [comment]
yep, see https://kube.rs/controllers/schemas/

**Comment by ivaltryek** [comment]
Gotcha!, Thanks
